controllers/envoy_local_limit.go

```diff
@@ -332,18 +337,28 @@ func generateEnvoyLocalRateLimitEnforced() *envoy_core_v3.RuntimeFractionalPerce
 }
 
 func generateEnvoyVhostMatch(vhostName string) *networking.EnvoyFilter_EnvoyConfigObjectMatch {
-	return &networking.EnvoyFilter_EnvoyConfigObjectMatch{
+	match := &networking.EnvoyFilter_EnvoyConfigObjectMatch{
+		Context: networking.EnvoyFilter_SIDECAR_INBOUND,
 		ObjectTypes: &networking.EnvoyFilter_EnvoyConfigObjectMatch_RouteConfiguration{
 			RouteConfiguration: &networking.EnvoyFilter_RouteConfigurationMatch{
 				Vhost: &networking.EnvoyFilter_RouteConfigurationMatch_VirtualHostMatch{
-					Name: vhostName,
 					Route: &networking.EnvoyFilter_RouteConfigurationMatch_RouteMatch{
 						Name: "default",
 					},
 				},
 			},
 		},
 	}
+	if vhostName != model.AllowAllPort {
```

Comment:
we can use the implicit sematic "empty means match-all" which is used across the istio code base.